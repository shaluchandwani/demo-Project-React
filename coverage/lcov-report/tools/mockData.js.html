<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for tools/mockData.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">tools</a> mockData.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>4/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>4/4</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a>
<a name='L55'></a><a href='#L55'>55</a>
<a name='L56'></a><a href='#L56'>56</a>
<a name='L57'></a><a href='#L57'>57</a>
<a name='L58'></a><a href='#L58'>58</a>
<a name='L59'></a><a href='#L59'>59</a>
<a name='L60'></a><a href='#L60'>60</a>
<a name='L61'></a><a href='#L61'>61</a>
<a name='L62'></a><a href='#L62'>62</a>
<a name='L63'></a><a href='#L63'>63</a>
<a name='L64'></a><a href='#L64'>64</a>
<a name='L65'></a><a href='#L65'>65</a>
<a name='L66'></a><a href='#L66'>66</a>
<a name='L67'></a><a href='#L67'>67</a>
<a name='L68'></a><a href='#L68'>68</a>
<a name='L69'></a><a href='#L69'>69</a>
<a name='L70'></a><a href='#L70'>70</a>
<a name='L71'></a><a href='#L71'>71</a>
<a name='L72'></a><a href='#L72'>72</a>
<a name='L73'></a><a href='#L73'>73</a>
<a name='L74'></a><a href='#L74'>74</a>
<a name='L75'></a><a href='#L75'>75</a>
<a name='L76'></a><a href='#L76'>76</a>
<a name='L77'></a><a href='#L77'>77</a>
<a name='L78'></a><a href='#L78'>78</a>
<a name='L79'></a><a href='#L79'>79</a>
<a name='L80'></a><a href='#L80'>80</a>
<a name='L81'></a><a href='#L81'>81</a>
<a name='L82'></a><a href='#L82'>82</a>
<a name='L83'></a><a href='#L83'>83</a>
<a name='L84'></a><a href='#L84'>84</a>
<a name='L85'></a><a href='#L85'>85</a>
<a name='L86'></a><a href='#L86'>86</a>
<a name='L87'></a><a href='#L87'>87</a>
<a name='L88'></a><a href='#L88'>88</a>
<a name='L89'></a><a href='#L89'>89</a>
<a name='L90'></a><a href='#L90'>90</a>
<a name='L91'></a><a href='#L91'>91</a>
<a name='L92'></a><a href='#L92'>92</a>
<a name='L93'></a><a href='#L93'>93</a>
<a name='L94'></a><a href='#L94'>94</a>
<a name='L95'></a><a href='#L95'>95</a>
<a name='L96'></a><a href='#L96'>96</a>
<a name='L97'></a><a href='#L97'>97</a>
<a name='L98'></a><a href='#L98'>98</a>
<a name='L99'></a><a href='#L99'>99</a>
<a name='L100'></a><a href='#L100'>100</a>
<a name='L101'></a><a href='#L101'>101</a>
<a name='L102'></a><a href='#L102'>102</a>
<a name='L103'></a><a href='#L103'>103</a>
<a name='L104'></a><a href='#L104'>104</a>
<a name='L105'></a><a href='#L105'>105</a>
<a name='L106'></a><a href='#L106'>106</a>
<a name='L107'></a><a href='#L107'>107</a>
<a name='L108'></a><a href='#L108'>108</a>
<a name='L109'></a><a href='#L109'>109</a>
<a name='L110'></a><a href='#L110'>110</a>
<a name='L111'></a><a href='#L111'>111</a>
<a name='L112'></a><a href='#L112'>112</a>
<a name='L113'></a><a href='#L113'>113</a>
<a name='L114'></a><a href='#L114'>114</a>
<a name='L115'></a><a href='#L115'>115</a>
<a name='L116'></a><a href='#L116'>116</a>
<a name='L117'></a><a href='#L117'>117</a>
<a name='L118'></a><a href='#L118'>118</a>
<a name='L119'></a><a href='#L119'>119</a>
<a name='L120'></a><a href='#L120'>120</a>
<a name='L121'></a><a href='#L121'>121</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">const Gallery = [
  {
    id: 1,
    title: "Pepe Jeans",
    slug: "Pepe-Jeans",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "200$",
    image:
      "https://assets.myntassets.com/dpr_2,q_60,w_210,c_limit,fl_progressive/assets/images/8544287/2019/2/12/66467281-c42e-4177-a0c1-8826ab1ac2371549966081210-Pepe-Jeans-Men-Blue-Slim-Fit-Low-Rise-Clean-Look-Stretchable-1.jpg"
  },
  {
    id: 2,
    title: "Micheal Chors Handbag",
    slug: "Micheal-Chors-Handbag",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "500$",
    image:
      "https://images-na.ssl-images-amazon.com/images/I/81UYkEoyIIL._UY500_.jpg"
  },
  {
    id: 3,
    title: "Soft Toy Stitch by Disney",
    slug: "Soft-Toy-Stitch-by-Disney",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "1000$",
    image: "https://images-na.ssl-images-amazon.com/images/I/51Mt0yxK66L.jpg"
  },
  {
    id: 4,
    title: "Indian Wear Kerela Saree",
    slug: "Indian-Wear-Kerela-Saree",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "2000$",
    image:
      "https://4.imimg.com/data4/OO/WU/ANDROID-45970894/product-500x500.jpeg"
  },
  {
    id: 5,
    title: "Men's Tshirt",
    slug: "Men's-Tshirt",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "2000$",
    image:
      "https://ae01.alicdn.com/kf/HTB144XeKVXXXXbkXXXXq6xXFXXXB/official-DJ-T-Shirts-Men-Short-Sleeve-Man-T-Shirt-Cotton-O-Neck-The-New-Cheap.jpg_640x640.jpg"
  },
  {
    id: 6,
    title: "Female Hair Band",
    slug: "Female-Hair-Band",
    vender: 3,
    size: ["medium", "small", "large"],
    amount: "2$",
    image:
      "https://ae01.alicdn.com/kf/HTB1hX3dmvBNTKJjy0Fdq6APpVXa2/Women-Turban-Twist-Headband-Head-Wrap-Twisted-Knotted-Knot-Hair-Band-Candy-Hot-Sale-L4-B3.jpg"
  },
  {
    id: 7,
    title: "Multiutility Bag",
    slug: "Multiutility-Bag",
    vender: 1,
    size: ["medium", "small", "large"],
    amount: "700$",
    image: "http://happyjute.com/wp-content/uploads/2017/03/p1-4.jpg"
  },
  {
    id: 8,
    title: "Roadster Jeans",
    slug: "Roadster-Jeans",
    vender: 2,
    size: ["medium", "small", "large"],
    amount: "200$",
    image:
      "https://surbhi.bitumish.in/wp-content/uploads/2019/01/download_4-e1547990689950.jpg"
  },
  {
    id: 9,
    title: "HeadPhones JBL",
    slug: "HeadPhones-JBL",
    vender: 1,
    size: ["Regular"],
    amount: "5000$",
    image:
      "https://images-na.ssl-images-amazon.com/images/I/71h2HWoQBEL._SX569_.jpg"
  },
  {
    id: 10,
    title: "Speaker JBL BoomBox",
    slug: "Speaker-JBL-BoomBox",
    vender: 3,
    size: ["Regular"],
    amount: "10000$",
    image:
      "https://pisces.bbystatic.com/image2/BestBuy_US/images/products/5968/5968903_sd.jpg;maxHeight=640;maxWidth=550"
  }
];
&nbsp;
const venders = [
  { id: 1, name: "Allen Solly creations" },
  { id: 2, name: "Adhya Creations" },
  { id: 3, name: "SHEIN" }
];
&nbsp;
const newGallery = {
  id: null,
  title: "",
  vender: null,
  size: "",
  amount: "$"
};
&nbsp;
module.exports = {
  newGallery,
  Gallery,
  venders
};
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Aug 09 2019 15:23:19 GMT+0200 (Central Africa Time)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
<script src="../block-navigation.js"></script>
</body>
</html>
