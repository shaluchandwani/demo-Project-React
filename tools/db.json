{
  "Gallery": [
    {
      "id": 2,
      "title": "Micheal Chors Handbag",
      "slug": "Micheal-Chors-Handbag",
      "vender": 1,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "500$",
      "image": "https://images-na.ssl-images-amazon.com/images/I/81UYkEoyIIL._UY500_.jpg"
    },
    {
      "id": 3,
      "title": "Soft Toy Stitch by Disney",
      "slug": "Soft-Toy-Stitch-by-Disney",
      "vender": 1,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "1000$",
      "image": "https://images-na.ssl-images-amazon.com/images/I/51Mt0yxK66L.jpg"
    },
    {
      "id": 4,
      "title": "Indian Wear Kerela Saree",
      "slug": "Indian-Wear-Kerela-Saree",
      "vender": 1,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "2000$",
      "image": "https://4.imimg.com/data4/OO/WU/ANDROID-45970894/product-500x500.jpeg"
    },
    {
      "id": 5,
      "title": "Men's Tshirt",
      "slug": "Men's-Tshirt",
      "vender": 1,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "2000$",
      "image": "https://ae01.alicdn.com/kf/HTB144XeKVXXXXbkXXXXq6xXFXXXB/official-DJ-T-Shirts-Men-Short-Sleeve-Man-T-Shirt-Cotton-O-Neck-The-New-Cheap.jpg_640x640.jpg"
    },
    {
      "id": 6,
      "title": "Female Hair Band",
      "slug": "Female-Hair-Band",
      "vender": 3,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "2$",
      "image": "https://ae01.alicdn.com/kf/HTB1hX3dmvBNTKJjy0Fdq6APpVXa2/Women-Turban-Twist-Headband-Head-Wrap-Twisted-Knotted-Knot-Hair-Band-Candy-Hot-Sale-L4-B3.jpg"
    },
    {
      "id": 7,
      "title": "Multiutility Bag",
      "slug": "Multiutility-Bag",
      "vender": 1,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "700$",
      "image": "http://happyjute.com/wp-content/uploads/2017/03/p1-4.jpg"
    },
    {
      "id": 8,
      "title": "Roadster Jeans",
      "slug": "Roadster-Jeans",
      "vender": 2,
      "size": [
        "medium",
        "small",
        "large"
      ],
      "amount": "200$",
      "image": "https://surbhi.bitumish.in/wp-content/uploads/2019/01/download_4-e1547990689950.jpg"
    },
    {
      "id": 9,
      "title": "HeadPhones JBL",
      "slug": "HeadPhones-JBL",
      "vender": 1,
      "size": [
        "Regular"
      ],
      "amount": "5000$",
      "image": "https://images-na.ssl-images-amazon.com/images/I/71h2HWoQBEL._SX569_.jpg"
    },
    {
      "id": 10,
      "title": "Speaker JBL BoomBox",
      "slug": "Speaker-JBL-BoomBox",
      "vender": 3,
      "size": [
        "Regular"
      ],
      "amount": "10000$",
      "image": "https://pisces.bbystatic.com/image2/BestBuy_US/images/products/5968/5968903_sd.jpg;maxHeight=640;maxWidth=550"
    },
    {
      "id": 11,
      "title": "Men's Tshirt small",
      "vender": 2,
      "size": "regular",
      "amount": "20$",
      "createdAt": 1565356701567,
      "slug": "men-s-tshirt-small"
    }
  ],
  "venders": [
    {
      "id": 1,
      "name": "Allen Solly creations"
    },
    {
      "id": 2,
      "name": "Adhya Creations"
    },
    {
      "id": 3,
      "name": "SHEIN"
    }
  ]
}