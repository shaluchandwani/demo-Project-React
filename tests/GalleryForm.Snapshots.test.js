import React from "react";
import GalleryOneForm from "../src/components/Gallery/GalleryForm";
import renderer from "react-test-renderer";
import { Gallery, venders } from "../tools/mockData";

it("sets submit button label 'Saving...' when saving is true", () => {
  const tree = renderer.create(
    <GalleryOneForm
      GalleryOne={Gallery[0]}
      venders={venders}
      onSave={jest.fn()}
      onChange={jest.fn()}
      saving
    />
  );

  expect(tree).toMatchSnapshot();
});

it("sets submit button label 'Save' when saving is false", () => {
  const tree = renderer.create(
    <GalleryOneForm
      GalleryOne={Gallery[0]}
      venders={venders}
      onSave={jest.fn()}
      onChange={jest.fn()}
      saving={false}
    />
  );

  expect(tree).toMatchSnapshot();
});
