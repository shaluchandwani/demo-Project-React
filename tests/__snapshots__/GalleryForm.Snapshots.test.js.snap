// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sets submit button label 'Save' when saving is false 1`] = `
<form
  onSubmit={[MockFunction]}
>
  <h2>
    Edit
     Items in Gallery
  </h2>
  <div
    className="form-group"
  >
    <label
      htmlFor="title"
    >
      Title
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="title"
        onChange={[MockFunction]}
        type="text"
        value="Pepe Jeans"
      />
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="vender"
    >
      Vender Name
    </label>
    <div
      className="field"
    >
      <select
        className="form-control"
        name="vender"
        onChange={[MockFunction]}
        value={1}
      >
        <option
          value=""
        >
          Select Vender
        </option>
        <option
          value={1}
        >
          Allen Solly creations
        </option>
        <option
          value={2}
        >
          Adhya Creations
        </option>
        <option
          value={3}
        >
          SHEIN
        </option>
      </select>
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="size"
    >
      Size
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="size"
        onChange={[MockFunction]}
        type="text"
        value={
          Array [
            "medium",
            "small",
            "large",
          ]
        }
      />
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="amount"
    >
      amount
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="amount"
        onChange={[MockFunction]}
        type="text"
        value="200$"
      />
    </div>
  </div>
  <button
    className="btn btn-primary"
    disabled={false}
    type="submit"
  >
    Save
  </button>
</form>
`;

exports[`sets submit button label 'Saving...' when saving is true 1`] = `
<form
  onSubmit={[MockFunction]}
>
  <h2>
    Edit
     Items in Gallery
  </h2>
  <div
    className="form-group"
  >
    <label
      htmlFor="title"
    >
      Title
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="title"
        onChange={[MockFunction]}
        type="text"
        value="Pepe Jeans"
      />
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="vender"
    >
      Vender Name
    </label>
    <div
      className="field"
    >
      <select
        className="form-control"
        name="vender"
        onChange={[MockFunction]}
        value={1}
      >
        <option
          value=""
        >
          Select Vender
        </option>
        <option
          value={1}
        >
          Allen Solly creations
        </option>
        <option
          value={2}
        >
          Adhya Creations
        </option>
        <option
          value={3}
        >
          SHEIN
        </option>
      </select>
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="size"
    >
      Size
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="size"
        onChange={[MockFunction]}
        type="text"
        value={
          Array [
            "medium",
            "small",
            "large",
          ]
        }
      />
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="amount"
    >
      amount
    </label>
    <div
      className="field"
    >
      <input
        className="form-control"
        name="amount"
        onChange={[MockFunction]}
        type="text"
        value="200$"
      />
    </div>
  </div>
  <button
    className="btn btn-primary"
    disabled={true}
    type="submit"
  >
    Saving...
  </button>
</form>
`;
